### DISCUSSION TITLE
how can I organize torrents by tracker?

### DISCUSSION THREAD

**Discussion by gchingon** [tfeldmann/organize#196] [Q&A]
currently I have `organize` config set to move torrent files to a folder, but I'd like to sort them into subfolders. Ideally based on filetype listed in the tracker so that:
`iso.torrent` goes into `/path/to/iso/ObscureLinuxDistroIso.torrent`
`publicDomainMovie` into `/path/to/movie/publikDomainMovieName.torrent`

otherwise is it possible to sort by tracker name?

`*publicDomainTracker.net*` move to `/path/to/pubDom/`
`*secretPrivateTracker.me` move to `/path/to/priTrak/`

**Comment by tfeldmann** [comment]
Unfortunately parsing torrent files is built in at the moment. You'd have to create your own `python` filter where you parse the torrent (using the `torrent_parser` python package or similiar) and then move the files according to that.
